docs/lib/datastore/getting-started.md

```diff
@@ -9,44 +9,155 @@ Amplify DataStore provides a programming model for leveraging shared and distrib
 
 <amplify-callout>
 
-DataStore can be used as a local-only data persistence mechanism and an AWS account is not required for that. However, if you wish to sync with the cloud it is recommended you [Install and configure the Amplify CLI](~/cli/start/install.md).
+**Note:** cloud synchronization is optional and disabled by default. This allows you to start with DataStore right away, even without an AWS account.
```

Comment:
* Not sure who updated it but, I love the intro where it says "Amplify DataStore provides a programming model for leveraging shared..."

* This allows you to start persisting data locally to your device with DataStore, even without an AWS account.